Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:16-alpine3.13
```

Comment:
I would probably use a [build ARG](https://docs.docker.com/engine/reference/builder/#arg) this will allow us to build multiple images tagged with the node version.

Reply:
I've added a build ARG for node version.